scala-ide-scaps-plugin/src/main/scala/scaps/eclipse/ui/handlers/SearchUCHandler.scala

```diff
@@ -9,11 +9,12 @@ import scaps.api.ValueDef
 import scaps.eclipse.core.services.ScapsSearchService
 
 object SearchUCHandler {
+  // Wozu?
   private def INSTANCE = new SearchUCHandler(ScapsService.SEARCH)
   def apply(): SearchUCHandler = INSTANCE
 }
 
-class SearchUCHandler(private val scapsSearchService: ScapsSearchService) {
+class SearchUCHandler(scapsSearchService: ScapsSearchService) {
```

Comment:
Konstruktorparameter in normalen Klassen sind eigentlich dasselbe wie private vals, also nur innerhalb der Klasse sichtbare Variablen.